continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -2,5 +2,5 @@ python:
     - 3.8
     - 3.9
 cython_enabled:
-    - True   # [linux]
-    - False
+    - true   # [linux]
+    - false
```

Comment:
This shouldn't matter, but maybe I'm missing something

https://yaml.org/type/bool.html

Reply:
Yeah that's what I figured, made the change because the syntax highlighting on GitHub's end made it seem like the two variants were being parsed differently

Reply:
Ah ok. Good to know. Though a bit odd they make it to Jinja without being interpreted as `bool`s here first